modules/ROOT/pages/teams.adoc

```diff
@@ -0,0 +1,59 @@
+
+After your child teams are created, you can invite new users directly to the appropriate team using email invitations or using the xref:configure-teams#invite-users-with-teams[Teams feature interface].
+When you add a user to a team, you can also specify whether they should be a _team maintainer_ (a user who can manage team membership) or a _member_ (a user who receives the permissions outlined in the team structure). Outside of teams, you can still xref:user-permissions.adoc#grant-permissions-to-an-individual-user.[assign specific permissions to individual users] if they require permissions that should not extend to an entire team.
+
+Adopting a team structure in your organization also enables you to share assets more easily. In Anypoint Exchange, you can share an asset with an entire team rather than having to select each user individually.
+
+[NOTE]
+To manage teams permissions, you must have the Organization Administrator permission over the business group that contains resources that you want to give access over to a specific user or a team.
+You can modify any permissions for all teams in your organization only if you have the Organization Administrator permission over the whole organization.
+
+== Permissions Structure
+
+When you are planning your organization’s team structure, consider the following:
+
+* Each member of your organization is a member of the “Everyone at <Your Organization> team.
+* Your team structure should reflect the structure of your organization so that you can plan your teams and configure permissions according to member needs.
+* Teams names must be unique across your organization.
+* Your organization can have up to 2000 teams.
+* Your teams can be nested up to 10 levels, including the “Everyone at <Your Organization> team.
+* You can create teams without adding members.
+* Every child team inherits permissions from its parent team.
+
+When you configure permissions in your organization, you should give your organization’s main team, “Everyone at <Your Organization>, permissions that do not afford too many privileges. Since every member of your organization is a member of that team, you should only configure permissions that you want every user to have. The child teams you create under the top level team inherit the permissions of the parent team. You can give each child team more specialized permissions that reflects the team members’ responsibilities.
+
+Each team has a team maintainer who can add new users to their own team. If you have the Administrator permission over your master organization, you can add users to any team and select whether they are members or maintainers. A team maintainer can only assign members to their specific team or child teams. Team maintainers can also designate new maintainers for teams they manage.
+
+Any user can view their team’s hierarchy in the *Teams* section in Access Management.
```

Comment:
Any user can view this data in the API, but we only plan on opening Access Management up to maintainers.